app/controllers/index.rb

```diff
@@ -0,0 +1,74 @@
+get '/' do
+  # Look in app/views/index.erb
+  redirect '/login'
+end
+
+# login and signup page
+get '/login' do
+  erb :login
+end
+
+#POST login to existing account
+post '/login' do
+  user = User.find_by(user_name: params[:user_name])
+    if user.authenticate(params[:password])
```

Comment:
Calling `user.authenticate` assumes that a user was found in the database -- need to check whether `user` exists before that!